docs/using-the-jdbc-driver/SupportForRDSMultiAzDBCluster.md

```diff
@@ -0,0 +1,55 @@
+# Enhanced Support for Amazon RDS Multi-AZ DB Cluster
+
+As of [v2.3.0](https://github.com/awslabs/aws-advanced-jdbc-wrapper/releases/tag/2.3.0), the AWS JDBC Driver has expanded its support for Amazon RDS Multi-AZ DB Cluster Deployment. By leveraging the topology information within the RDS Multi-AZ DB Cluster, the driver is capable of switching over the connection to a new writer node in approximately 1 second or less, given there is no replica lag during minor version upgrades or OS maintenance upgrades.
+
+## General Usage
+
+The process of using the AWS JDBC Driver with RDS Multi-AZ DB Cluster is the same as using it with an RDS Aurora cluster. All properties, configurations, functions, etc., remain consistent. Instead of connecting to a generic database endpoint, simply replace the endpoint with the Cluster Writer Endpoint provided by the RDS Multi-AZ DB Cluster.
+
+### MySQL
+
+To prepare the connection:
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:aws-wrapper:mysql://cluster-writer-endpoint[:port]/database", props);
+```
+
+### PostgreSQL
+
+The topology information is populated in Amazon RDS for PostgreSQL versions 13.12, 14.9, 15.4, or higher, starting from revision R2. Ensure you have a supported PostgreSQL version deployed.
+
+Per AWS documentation and [this blog post](https://aws.amazon.com/blogs/database/achieve-one-second-or-less-downtime-with-the-advanced-jdbc-wrapper-driver-when-upgrading-amazon-rds-multi-az-db-clusters/), the `rds_tools` extension must be manually installed using the following DDL before the topology information becomes available:
+
+```sql
+CREATE EXTENSION rds_tools;
+```
+
+Then, prepare the connection with:
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:aws-wrapper:postgresql://cluster-writer-endpoint[:port]/database", props);
+```
+
+
+## Optimizing Switchover Time
```

Comment:
Let's properly define what we mean by switchover if we want to talk about it

Reply:
RDS Multi-AZ DB Cluster prefers terminology `switchover` when referring the "fast failover" feature and it's been used everywhere.

Reply:
The feature announcement is out at https://aws.amazon.com/about-aws/whats-new/2023/11/amazon-rds-multi-az-two-stanbys-upgrades-downtime/

Added to the doc as well.